src/components/app.tsx

```diff
@@ -0,0 +1,64 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import Header from "./header"
+import Form from "./form"
+import Canvas from "./canvas"
+import * as React from 'react'
+import { Furniture } from '../types/furniture'
+import config from '../utils/config'
+
+interface AppState {
+  canvasBlob?: Blob
+  furniture?: Furniture
+  originalImageData?: object
+}
+
+
+class App extends React.Component<any, AppState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
```

Comment:
Could this initial state be pushed to an exported object within `utils`? It should make it easier to understand.

Reply:
Yeah good point, I can make that change.